packages/lit-dev-content/site/docs/v3/tools/testing.md

```diff
@@ -75,3 +75,49 @@ export default {
 };
 ```
 
+## Using WebdriverIO
+
+[WebdriverIO](https://webdriver.io) is a good option for your component or end-to-end tests. It has very compelling advantages like support for [mocking](https://webdriver.io/docs/component-testing/mocking) and [code coverage](https://webdriver.io/docs/component-testing/coverage) reporting.
+
+You can set up WebdriverIO in your project via:
+
+```bash
+npm init wdio@latest ./
+```
+
+It will start a configuration wizard that will guide you through some questions. Make sure select the following:
+
+- __What type of testing would you like to do?__:<br>Component or Unit Testing - in the browser
+- __Which framework do you use for building components?__:<br>Lit
```

Comment:
Looking at the preview URL, I think I'd prefer the removal of the colons. Currently there is double punctuation. e.g. `?:`. Maybe change to only the question mark for both items.

Reply:
I was debating whether this part should also be inside a codeblock since it's the selection with the CLI.

Reply:
IMHO it is visually more clear to keep the list
<img width="810" alt="Screenshot 2024-01-08 at 4 15 00 PM" src="https://github.com/lit/lit.dev/assets/731337/a067ea49-d8ce-440b-bb6d-f28e5707359e">

vs

<img width="807" alt="Screenshot 2024-01-08 at 4 15 53 PM" src="https://github.com/lit/lit.dev/assets/731337/9216234d-7f5d-4261-87a4-9032fca0d58b">

Happy with either, please advise!

Reply:
Indeed, without color, the cli output in code block is less clear. let's keep the list but just without the ":" following the "?" per Andrew's suggestion. Thanks for the screenshots!

Reply:
removed colons 👍